app.js

```diff
@@ -29,6 +29,12 @@ const appRouter = express.Router();
 const adminApp = express();
 const adminAppRouter = express.Router();
 
+// Body will not be found in request unless bodyParser is used
+app.use(bodyParser.urlencoded({
+  extended: true,
+}));
+app.use(bodyParser.json());
+
```

Comment:
This issue should be handled properly if we specify the `consumes` type in openapi file. Example: https://github.com/osu-mist/notes-api/blob/master/openapi.yaml#L249-L280

Also, can you make sure the `required` fields are all added and `additionalProperties` are set correctly for the properties in all post body?

Reply:
added consume and check all post endpoints for `required` and `additionalProperties` fields.